src/cpp/client/channel_cc.cc

```diff
@@ -50,8 +50,16 @@
 namespace grpc {
 
 static internal::GrpcLibraryInitializer g_gli_initializer;
-Channel::Channel(const grpc::string& host, grpc_channel* channel)
+Channel::Channel(
+    const grpc::string& host, grpc_channel* channel,
+    std::unique_ptr<std::vector<
+        std::unique_ptr<experimental::ClientInterceptorFactoryInterface>>>
+        interceptor_creators)
     : host_(host), c_channel_(channel) {
+  auto vector = interceptor_creators.release();
```

Comment:
I like `auto*` so that the reader understands right away that this is a pointer.

Reply:
done